deployment/ansible/raster-foundry.yml

```diff
@@ -15,6 +15,6 @@
     - name: Change directory on login
       lineinfile: dest=/home/vagrant/.bashrc regexp=^raster-foundry_PROFILE line="cd /opt/raster-foundry/"
 
-    - name: Set Environment variable for AWS profile
-      lineinfile: dest=/etc/environment regexp=^AWS_PROFILE line="AWS_PROFILE={{aws_profile}}"
```

Comment:
The only thing to consider here is if we use `AWS_PROFILE` in the context of `sudo`. Keeping the profile defined in `.bashrc` won't populate the `root` environment by default.